src/res/translation/translation_de_DE.ts

```diff
@@ -1477,7 +1477,7 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="271"/>
         <source>Select the meter style to be used for the level meters. The Bar (narrow) and LEDs (round, small) options only apply to the mixerboard. When Bar (narrow) is selected, the input meters are set to Bar (wide). When LEDs (round, small) is selected, the input meters are set to LEDs (round, big). The remaining options apply to the mixerboard and input meters.</source>
-        <translation type="unfinished"></translation>
+        <translation>Wähle den Stil aus, der für die Pegel-Messer benutzt wird. Die Stile Balken (schmal) und LEDs (rund, klein) beziehen sich nur auf den Mixerbereich. Die Auswahl des Stils Balken (schmal) sorgt dafür, dass die Eingangspegelanzeigen dennoch mit breiten Balken dargestellt werden. Wenn LEDs (rund, klein) ausgwählt sind, entsprechen die Eingangspegelanzeigen dem Stil LEDs (rund, groß). Die anderen Optionen beziehen sich sowohl auf den Mixer als auch auf die Eingangspegelanzeigen.</translation>
```

Comment:
I‘d put the style names in „ ”

Reply:
Done. I was a bit hesitant because the same could be said about English, I guess. ;)

Reply:
Yes. You’re right.